src/natcap/invest/coastal_vulnerability.py

```diff
@@ -1273,17 +1273,18 @@ def extract_bathymetry_along_ray(
         iy = int((point.y - bathy_gt[3]) / bathy_gt[5])
         win_size = 1
 
-        value = bathy_band.ReadAsArray(
+        values = bathy_band.ReadAsArray(
             xoff=ix, yoff=iy,
             win_xsize=win_size, win_ysize=win_size)
-        if value is None:
+        if values is None:
             raise ValueError(
                 'got a %s when trying to read bathymetry at %s. Does the '
                 'bathymetry input fully cover the fetch ray area?'
-                % (value, {'xoff': ix, 'yoff': iy,
+                % (values, {'xoff': ix, 'yoff': iy,
                            'win_xsize': win_size, 'win_ysize': win_size}))
-        if ~numpy.isclose(value, bathy_nodata):
-            bathy_values.append(value)
+
+        if bathy_nodata is None or ~numpy.isclose(values, bathy_nodata):
+            bathy_values.append(values)

```

Comment:
Hey I'm sorry about the somewhat confusing chain of timings and comments over in the other thread.  You're right that we can't use `math.isclose` directly on the array, but since we know ahead of time that this will be a single value, we can just index into that one item, `values[0][0]` or the like: `not math.isclose(values[0][0])`

Reply:
Oh okay, that makes sense. The nodata value was also (incorrectly) a list so I fixed that in `calculate_wind_exposure`